src/main/java/org/bookstore/model/ShoppingCart.java

```diff
@@ -0,0 +1,45 @@
+package org.bookstore.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Entity
+@Data
+@Table(name = "shopping_cart")
```

Comment:
The name of database table should be in plural

Are you sure that liquibase didn't throw an error? Because in changeset you have another table name 

`shopping_carts`